src/libawkward/array/NumpyArray.cpp

```diff
@@ -2877,17 +3080,17 @@ namespace awkward {
                                                     ptr_lib_);
 
       if (mask) {
-        Index8 mask(outlength);
+        Index8 outmask(outlength);
         struct Error err = kernel::NumpyArray_reduce_mask_ByteMaskedArray_64(
           kernel::lib::cpu,   // DERIVE
-          mask.data(),
+          outmask.data(),
```

Comment:
reported by static analyser

Reply:
Yes, that overshadowed a variable of a different type (`bool` vs `Index8`) in the parent scope, which isn't wrong, but it definitely isn't good style. Changing the name resolves that, so this is good.